clash-cores/src/Clash/Cores/Xilinx/Ila/Internal.hs

```diff
@@ -148,15 +148,18 @@ areEqual :: Eq a => [a] -> Maybe a
 areEqual = \case { [x:_] -> Just x; _ -> Nothing } . group
 
 ilaBBF :: HasCallStack => BlackBoxFunction
-ilaBBF _isD _primName args _resTys = Lens.view tcCache >>= go
+ilaBBF _isD _primName args _resTys = do
+  instName <- fromMaybe "ila_inst" <$> Lens.view setName
```

Comment:
```suggestion
  instName <- fromMaybe "ila_inst" <$> traverse affixName (Lens.view setName)
```
The above really is just a suggestion. Normally we add suffixes and prefixes to `setName`. I know this is not what we want for ILAs, but now the ILA is the odd one out with regards to `setName`.

Reply:
No, I agree, this was a simple oversight. This is not the place to strip affixes; if we want to do that, we should do it elsewhere.